### DISCUSSION TITLE
希望可以用于真机

### DISCUSSION THREAD

**Discussion by zynetc** [LmeSzinc/AzurLaneAutoScript#2434] [General]
希望可以用于真机，目前好像分辨率不太匹配

**Comment by ec7529a2** [comment]
这里有文档，但是很麻烦，建议用模拟器
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/Emulator_cn#-%E5%9C%A8%E5%AE%89%E5%8D%93%E7%9C%9F%E6%9C%BA%E8%BF%90%E8%A1%8Calas
